docs/v107/README.md

```diff
@@ -94,7 +94,7 @@ When you got a request instance, you can make a call:
 var request = new RestRequest()
     .AddQueryParameter("foo", "bar")
     .AddJsonBody(someObject);
-var response = await client.PostAsync<MyResponse>(request, cancellationToken);
+var response = await client.PostJsonAsync<MyResponse>("path/to/endpoint",request, cancellationToken);
```

Comment:
Sorry but this is wrong. The second argument for `PostJsonAsync` is not a request, it's the payload.